packages/@o3r/design/src/core/design-token/renderers/design-token-style.renderer.ts

```diff
@@ -60,6 +62,9 @@ export const renderDesignTokens = async (variableSet: DesignTokenVariableSet, op
   await Promise.all(
     Object.entries(updates).map(async ([file, vars]) => {
       const styleContent = existsFile(file) ? await readFile(file) : '';
+      if (!existsFile(file) && !vars.length) {
```

Comment:
Probably better to avoid unnecessary disk access by storing the result of `existsFile(file)` (or using `!styleContent` instead maybe?)

Reply:
I will go with the intermediate var (to let the possibility of the file to be empty but still receive the shell content)